spec/move-tags-spec.js

```diff
@@ -0,0 +1,213 @@
+// var tidy = require('../mustache-tidy.js').mustacheTidy;
+var tidy = require('../');
+
+describe('Moving tags', function() {
```

Comment:
@Minstel You typically do only `describe`, with multiple `it` calls in one spec file.